kodilla-spring/src/main/java/com/kodilla/spring/portfolio/BoardConfig.java

```diff
@@ -0,0 +1,32 @@
+package com.kodilla.spring.portfolio;
+
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.context.annotation.Scope;
+
+@Configuration
+public class BoardConfig {
+
+    @Bean
+    public Board getBoard() {
+        return new Board(getToDoList(), getInProgressList(), getDoneList());
+    }
+
+    @Bean(name = "toDoList")
+    @Scope("prototype")
+    public TaskList getToDoList() {
```

Comment:
Lepiej jest nazwać metodę tak, jak beana, czyli `toDoList()`, zamiast dawać parametr w adnotacji `@Bean`

Reply:
tym bardziej, że te metody nie są getterami, a metody zaczynające się `get...` mamy zarezerwowane dla getterów - to jest mylące